internal/command/key_pair.go

```diff
@@ -34,3 +38,130 @@ func (c *Commands) GenerateSigningKeyPair(ctx context.Context, algorithm string)
 		privateKeyExp, publicKeyExp))
 	return err
 }
+
+func (c *Commands) GenerateSAMLCACertificate(ctx context.Context, algorithm string) error {
+	now := time.Now().UTC()
+	after := now.Add(c.certificateLifetime)
+	privateCrypto, publicCrypto, certificateCrypto, err := crypto.GenerateEncryptedKeyPairWithCACertificate(c.certKeySize, c.keyAlgorithm, c.certificateAlgorithm, &crypto.CertificateInformations{
+		SerialNumber: big.NewInt(int64(rand.Intn(50000))),
+		Organisation: []string{"ZITADEL"},
+		CommonName:   "ZITADEL SAML CA",
+		NotBefore:    now,
```

Comment:
technically we could include it into the config, as it is self signed I just defined it here some time ago, don't now how necessary it would be to make it configurable